XPlat/AzureMessaging/iOS/source/ApiDefinition.cs

```diff
@@ -284,7 +284,7 @@ public partial interface MSNotificationHubMessage
 		string Body { get; }
 
 		[Export("userInfo")]
-		NSDictionary<NSString, NSString> UserInfo { get; }
+		NSDictionary UserInfo { get; }
```

Comment:
This is a breaking change. Just need confirmation.
Also, the iOS version was not increased, so this will not actually leave the door.

Reply:
Yes, this is a breaking change due to a miscommunication on our team.  This should have always been an `NSDictonary *` instead of` NSDictionary<NSString *, NSString *> *`. Currently, this doesn't work, because it casts an `InvalidCastException` due to this bug

Reply:
Will update to 3.1.1 then, since this was an issue.